rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/data/JobResultImpl.java

```diff
@@ -54,6 +55,8 @@
 
     private Map<String, TaskResult> preciousResults;
 
+    private Map<String, Serializable> JobMap = null;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "JobMap" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)